Sudoku/constants.js

```diff
@@ -0,0 +1,7 @@
+const SQUARES = require('./locationMap.js')
+const BOARD_LENGTH = 81
+const NUMBERS_PER_SQUARE = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
```

Comment:
@mantinone Nope, just do that for sake of what I've learned from C++, been working on old projects, like to use lots of headers. I just never understood export and require well enough in the past to do it until now.